util/image_viewer/spotfinder_frame.py

```diff
@@ -811,7 +811,7 @@ def draw_resolution_rings(self, unit_cell=None, space_group=None):
                 update=False,
             )
 
-    def sum_images(self):
+    def stack_images(self):
         if self.params.sum_images > 1:
```

Comment:
Would it make sense to also change the command line parameter to `stack_images` (possibly keeping around `sum_images` with a deprecation warning for the time being)?

Reply:
... maybe, provided the people's national bikeshed committee are satisfied with the nomenclature of stacking the images?

Reply:
I don't know how to make one phil parameter alias another, and deprecate one or the other, otherwise I would have done this...

Reply:
I could replace the name and just break the cl api

Reply:
R.e. marking phil parameters as deprecated:
https://github.com/cctbx/cctbx_project/issues/452#issuecomment-604313748

I would either:
1. Rename `sum_images` to `stack_images` and re-add `sum_images` with `.deprecated` attribute, and ignore. That way it doesn't fail, but users get a warning.
2. Just rename and accept that very few people likely use the CLI to the image viewer.

Reply:
Pushed change to this effect, if you use old command line option you get 

```
Grey-Area 0 :) $ dials.image_viewer imported.expt strong.refl sum_images=10
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
Sorry: Unknown command line parameter definition: sum_images = 10
```

which I think is OK